### DISCUSSION TITLE
Use data buckets in response rules II

### DISCUSSION THREAD

**Discussion by Jerantony** [mockoon/mockoon#1335] [Ideas] [answered]
One can already use data buckets in repsonse rules by using template helpers in the rule value since #986. Which is great to dynamically set values which the mock expects for certains properties for a successful response.

But in order to easily determine the response code of the mock even for routes which expect no sepcial parameters, it would be great if one could choose the data buckets as the target of response rules. Then one could create rules as in the image below and there are surely more applications. I'd love to implement this 😁 
<img width="1489" alt="Screenshot 2024-03-10 174516" src="https://github.com/mockoon/mockoon/assets/88666775/22ffe399-d110-46cd-9b15-a8b2301f5800">

**Comment by 255kb** [comment, marked as answer]
Good idea! 

Feel free to open an issue and I will assign it to you and put it on the roadmap. 
You can take example on the recent [global var rule implementation](https://github.com/mockoon/mockoon/commit/32e71338fcf23085a0399c93ce346ab896962ac1), it should be quite straightforward.

**Reply by Jerantony** [reply]
Thanks, created #1337

**Reply by 255kb** [reply]
Thanks, I commented the issue and assigned it to you.
